.github/workflows/kind-e2e.yaml

```diff
@@ -35,6 +35,8 @@ jobs:
       working-directory: ./src/github.com/${{ github.repository }}
       env:
         KIND_VERSION: v0.11.1
```

Comment:
Might want to bump Kind to v0.12.0 while you are at it

Reply:
Yes, let me create follow-up PR